it/uk/gov/hmrc/economiccrimelevyregistration/RegistrationSubmissionISpec.scala

```diff
@@ -27,13 +27,12 @@ class RegistrationSubmissionISpec extends ISpecBase {
           .copy(processingDate = Instant.parse("2007-12-25T10:15:30.00Z"))
 
       val registrationDate   = LocalDate.now().format(DateTimeFormatter.ISO_LOCAL_DATE)
-      val liabilityStartDate = LocalDate.now().format(DateTimeFormatter.ISO_LOCAL_DATE)
 
       stubSubscribeToEcl(
         validRegistration.expectedEclSubscription.copy(subscription =
           validRegistration.expectedEclSubscription.subscription.copy(
             legalEntityDetails = validRegistration.expectedEclSubscription.subscription.legalEntityDetails
-              .copy(registrationDate = registrationDate, liabilityStartDate = liabilityStartDate)
+              .copy(registrationDate = registrationDate, liabilityStartDate = "2022-04-01")
```

Comment:
Would it make sense to have a central constant value that can be referenced everywhere?